docs/joss_paper/paper.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'PyUnfold: A Python package for iterative unfolding'
+tags:
+  - Python
+  - unfolding
+  - estimation
+  - statistics
+  - deconvolution
+  - inverse problems
+authors:
+  - name: James Bourbeau
+    orcid: 0000-0003-2164-7789
+    affiliation: "1"
+    email: james.bourbeau@icecube.wisc.edu
+  - name: Zigfried Hampel-Arias
+    orcid: 0000-0003-0253-9117
+    affiliation: "1, 2"
+    email: zhampel@wipac.wisc.edu
+affiliations:
+ - name: University of Wisconsin - Madison, Wisconsin, USA
+   index: 1
+ - name: IIHE, Université Libre de Bruxelles, Bruxelles, Belgium
+   index: 2
+date: 05 May 2018
+bibliography: paper.bib
+---
+
+# Summary
+
+PyUnfold is an extensible framework for the deconvolution of discrete probability 
+distributions via the iterative unfolding method described in [@agostini]. 
+This method falls into the general class of inverse problems, and is especially powerful 
+when no analytic form of the inverse function is explicitly available, instead requiring
+an estimate (e.g. via a finite amount of simulation) of the response function.
+Given that measured data comprise a finite sample, PyUnfold also implements the uncertainty
+contributions stated in [@adye2].
+
+
+The primary purpose of PyUnfold is to provide an unfolding toolkit for members of all 
+scientific disciplines in an easy-to-use package.
+For example, unfolding methods are commonly used in the analysis pipeline of the 
+high-energy physics (HEP) community.
+However, the deconvolution packages used in HEP maintain a strong dependence on the 
+ROOT data analysis framework [@root], which is almost exclusively used in HEP.
+Instead, PyUnfold is built on top of the Python scientific computing stack (i.e. NumPy,
+SciPy, and pandas), thus broadening its scope to a general scientific audience.
+
+
+Indeed, the method itself is data-agnostic, referring to the measurement process
+as the smearing of a set of true causes into a set of detectable effects.
+For example one could define as causes the true energy of a particle and the effects
+the measured energy of that particle in a detector.
+Another example might be a set of diseases (causes) and possible clinical symptoms (effects).
+So long as it is possible to encode estimable resolutions and biases connecting causes to 
+effects in a binned response matrix, one can perform a deconvolution with PyUnfold. 
+
+
+PyUnfold has been designed to be both easy to use for first-time users as well as 
+flexible enough for fine-tuning an analysis.
+Its main strength is its simplicity, requiring a single line to run via the ``iterative_unfold`` 
```

Comment:
I really think we should condense this section. Basically just saying PyUnfold easy to use and here is what it can do. 

> PyUnfold has been designed to be both easy to use for first-time users as well as flexible enough for fine-tuning an analysis. It provides support for the following:

Those interested in using it will learn about the ``iterative_unfold`` function when they look at the documentation.